openfecwebapp/templates/datatable.html

```diff
@@ -48,7 +47,14 @@
 {% if office %}
 <script>
 var context = {
-  office: '{{ office }}'
+  office: '{{ office }}',
+};
+</script>
+{% endif %}
+{% if form_type %}
+<script>
+var context = {
+  formType: '{{ form_type }}',
```

Comment:
Is the idea to have both of these properties be in the `context` object if the conditions allow for it?  It doesn't look like that'll be the case here, and I wonder if there may be a possibility of both conditional checks being true.  If that were to happen, `context` would be overwritten and only contain what is in the last `if` statement here.

If this is not the case, the extra comma at the end might be an issue, I thought JS complained about that after the last item in a literal array or object definition.